inference-engine/src/plugin_api/cpp_interfaces/impl/ie_infer_request_internal.hpp

```diff
@@ -59,10 +59,13 @@ class InferRequestInternal : virtual public IInferRequestInternal {
      * @brief Default common implementation for all plugins with checking input and output blobs before inference
      */
     void Infer() override {
+        ResetCancellationRequest();
         checkBlobs();
         InferImpl();
     }
 
+    virtual void ResetCancellationRequest() = 0;
```

Comment:
Do we need this separate function? It looks like CPU-specific. IMO, plugin can do all resets in `InferImpl` method.

Reply:
But in this case i would need to move "bool cancel" to this class?

Reply:
Why? Can't CPU plugin reset the `bool cancel` state at the begging of its `InferImpl` method body?

Reply:
or plugin  can  reset  cancel state right after condition check. Like if (cancel) { cancel = false; handle cancellation; }

Reply:
Did remove ResetCancellationRequest, plugin will be responsible to reset cancel state

Reply:
@Drizshko , to finally close with the review, you mentioned that you have removed the ResetCancellationRequest, but it is still in the code. I'm ok with that, but let's cover the initial concern from @vinograd47